tex/chProofs.tex

```diff
@@ -370,15 +396,17 @@ \subsection{Conditional statements}
 same as the one we have used in chapter~\ref{ch:prog} in order to
 represent function types. This is no accident, but we postpone further
 comments on the meaning of this arrow to section~\ref{sec:chi}. For
-now let us only stress that \C{->} associates on the right and that
+now let us only stress that \C{->} associates on the right, and
```

Comment:
OK, let me switch to "right-associative", which is easier to google for those who don't know it anyway. Commit incoming!